Documentation/security.md

```diff
@@ -0,0 +1,26 @@
+# Security best practices
+
+This document tries to give an overview of the security recommendations to follow when running rkt containers.
+
+## General recommendations
+
+* **Don't run containers as root.** Use the [user and group fields][aci-schema] in your images or the `--user` and `--group` [CLI flags][rkt-run-subcommands] when running containers.
```

Comment:
I'd suggest to rephrase this as "don't run applications as root". Random readers may misunderstand this and try to run rkt as non-root.